.travis.yml

```diff
@@ -18,13 +18,11 @@ matrix:
 addons:
   postgresql: '10'
 before_install:
-- source ${TRAVIS_BUILD_DIR}/tools/ci/before_install.sh
+- source ${TRAVIS_BUILD_DIR}/bin/ci/before_install.sh
 before_script:
-- bundle exec rake test:$TEST_SUITE:setup
-script:
-- bundle exec rake test:$TEST_SUITE
+- source ${TRAVIS_BUILD_DIR}/bin/ci/before_script.sh
 after_failure:
-- source ${TRAVIS_BUILD_DIR}/tools/ci/after_failure.sh
+- source ${TRAVIS_BUILD_DIR}/bin/ci/after_failure.sh
```

Comment:
One more minor thing, but now that these are a single item you don't need an Array, which slims up the file a bit.

Reply:
@Fryguy do you know we have tack on the `${TRAVIS_BUILD_DIR}` here where the other repos do not?

Reply:
we probably don't have to.  I think that's how Travis shows you how to do it in their documentation, and this is probably the first repo we did it with.

Reply:
:+1: I'll see about that and the bash script calling a ruby script in a follow-up, don't want to be changing too much when cross-repo /all takes hours to run :laughing: